projects/m1/033-shipping-calculator/js/index.js

```diff
@@ -0,0 +1,29 @@
+let ordersArray = [];
+
+function startInput() {
+   
+    let newOrder = "";
+
+    while (newOrder !== "n") {
+        newOrder = prompt("Inserisci un articolo nel carrello (inserisci n per terminare):");
+
+        if (newOrder !== "n") {
+            ordersArray.push(newOrder);
+        }
+    }
+}
+
+function definitiveCost(totalOrders) {
+    let basicCost = 10.99;
+    let otherCost = 2.99;
+    let totalCost = basicCost + (otherCost * totalOrders);
+    return totalCost;
+}
+
+function calculateAndShow() {
+    startInput(); 
+    const totalOrders = ordersArray.length; 
+    let totalshipping = definitiveCost(totalOrders); 
+    let ordersList = ordersArray.join(", "); 
+    document.getElementById('output').innerHTML = `Il costo totale della spedizione per gli articoli ${ordersList} è ${totalshipping.toFixed(2)}`;
+}
```

Comment:
Be careful using `let` and `const`.
Many statements are **incorrect**.
I recommend the following reading:
https://www.freecodecamp.org/italian/news/var-let-const-differenza/